resources/default-settings/mpd.default.conf

```diff
@@ -240,14 +240,12 @@ decoder {
 # autodetection for one sound card.
 #
 
-# Connect to Pulse locally via ALSA interface
+# Connect directly to PulseAudio
 audio_output {
-        type            "alsa"
-        name            "Global ALSA->Pulse stream"
-        mixer_type      "hardware"
-        mixer_control   "Master"
-        mixer_device    "pulse"
-        device          "pulse"
+        type            "pulse"
+        name            "mpd"
+        format          "48000:16:2"
+        mixer_type      "software"
```

Comment:
### **audio_output**
alsa should not be used at all
instead of:
	mpd -> alsa -> pulseaudio -> alsa
do directy:
	mpd -> pulseaudio -> alsa
you are right you don't need the mixer type and format line at all.
my asound.conf
```
pcm.pulse {
    type pulse
}

ctl.pulse {
    type pulse
}
```

### **user**
Same here - user-local service - but I did not find out yet why when mpd is started via socket it's always running as root!?

### general
As with all my suggestions (merge request) - it's just giving ideas what can be done or must be done in future. As you know I working on a yocto based version. Thank you!

Reply:
> Same here - user-local service - but I did not find out yet why when mpd is started via socket it's always running as root!?

That's an interesting point and should be investigated. Could you check the mpd.socket settings

~~~
sudo systemctl mpd.socket
systemctl --user mpd.socket
~~~

And do you have a test case to specifically trigger mpd via the socket?

Reply:
I have created a PR to fix the mpd.socket relationship during the installation process as that should definitely be fixed: #1707. #1707 It would still be good if you good give feedback it above stuff solves the issue.